DataFormats/Common/interface/RangeMap.h

```diff
@@ -77,7 +82,9 @@ namespace edm {
     /// to use only comparators provided with CMSSW release.
     template<typename CMP> 
     range get(ID id, CMP comparator) const {
+#ifndef __APPLE__
       using namespace __gnu_cxx;
```

Comment:
I don't know but the classes.h also has references to __gnu_cxx

input_line_9:66:3: error: use of undeclared identifier '__gnu_cxx'
  __gnu_cxx::new_allocator<char> dummy21;
  ^